Compiler/script2/cc_internallist.h

```diff
@@ -40,8 +40,8 @@ class SectionList
     {}
     SectionList(const std::vector<std::string> &&sections,
                 const std::map<size_t, size_t> &&off2sec)
-        : _sections(std::move(sections))
-        , _off2sec(std::move(off2sec))
+        : _sections(sections)
+        , _off2sec(off2sec)
```

Comment:
The way this is changed now defeats the purpose of this constructor, it was intended to "move" the passed objects, as opposed to making it copy.

The warning was probably because the constructor arguments are declared with "const", which is incorrect.
Should be:
```
SectionList(std::vector<std::string> &&sections,
                std::map<size_t, size_t> &&off2sec)
```

Reply:
OK, I'll fix it.